dvc/repo/ls.py

```diff
@@ -0,0 +1,99 @@
+import os
+
+
+@staticmethod
+def ls(url, targets=[], rev=None, recursive=None):
+    if os.path.exists(url):
```

Comment:
```suggestion
    if os.path.exists(url) and not rev:
```
so that revs work with local repos.

Reply:
@efiop what is the reason of this check? I think it's better to ignore  `rev` implicitly for the local path or throw an error  for invocation with local path with `rev`